manifests/metallb-frr.yaml

```diff
@@ -446,6 +446,9 @@ spec:
           emptyDir: {}
         - name: metrics
           emptyDir: {}
+        - name: frr-corefile
```

Comment:
These changes will be translated in the operator's bindata and it doesn't feel right.
Can we adjust the path under `/etc/frr` so we don't need to touch the manifests?

Same for ulimit below.

Reply:
the volume can be changed but we have to make sure we set RLIMIT for core to unlimiited not sure that can be done anywhere else outside the container init script here

Reply:
You can kubectl exec into the pod for example, still from CI and before running the tests.
Also, the issues we had on CI were related to the external frrs, the container inside the pods never showed the issue so far.

Reply:
I was adding this capability to production FRR pod so if it crash in production we need to have core file

Reply:
so it's not CI only? It feels weird that we need to set something explicit in the container, I wonder how the other components / platform handle this (i.e. ovn, to mention one).
We are not the only one having this problem, so it's probably worth investigating if it's only because we are running kind.

Reply:
i checked on real OCP cluster and the core pattern and ulimit has been set so yeah no need to change manifests so I limited this change just for CI components